content/security/committers.md

```diff
@@ -54,48 +54,51 @@ security nature of the commit.
 
 ### Report
 
-1. The person discovering the issue, the _reporter_, reports the
-vulnerability privately to `security@project.apache.org` or to
-`security@apache.org`.
+1. The person discovering the issue, the _reporter_, identifies the mailing list to report to.
+If the project the issue relates to has a security@*[project]*.apache.org mailing list listed on
+[security projects list](https://security.apache.org/projects/), use that mailing list. Otherwise, use `security@apache.org`.
 
-2. The Security team ignores any message to this inbox that does not relate to reporting or managing an
+2. Note that the Security teams will ignore all messages that do not relate to reporting or managing an
 undisclosed security vulnerability in Apache software.
 
 3. If the report comes to `security@apache.org`, the security team forwards
 it to the project's security list or, if the project does not
 have a security list, to the project's private (PMC) mailing list.
 The Security Team responds to the original reporter that they have done this.
 
+4. If the project does not have a dedicated `security@project.apache.org`
```

Comment:
Please do not add new numbered sections as this causes all the numbers to change and those numbers are used in various guides internally by projects and on our cveprocess tool.  I'm not sure this section is really needed either.

Reply:
I put the numbers back so the 'api' doesn't change :)

The text had come from lower down in the page (the lost sentences), but happy to delete if you feel it's unnecessary.